src/main/java/com/yurjinia/user/service/UserProfileService.java

```diff
@@ -102,4 +123,10 @@ private UserProfileEntity getUserProfileByEmail(String email) {
                 .orElseThrow(() -> new CommonException(ErrorCode.USER_NOT_FOUND, HttpStatus.NOT_FOUND));
     }
 
+    private void updateFieldIfNotBlank(String newValue, Consumer<String> updater) {
```

Comment:
good implementation with Consumers!

try to come up with more descriptive name, because this one is generic and not aligns with the business logic.